magellan-compose/src/main/java/com/example/magellan/compose/ComposeInterop.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.magellan.compose
+
+import android.view.View
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.viewinterop.AndroidView
+import com.wealthfront.magellan.core.Displayable
+import com.wealthfront.magellan.core.Navigable
+import com.wealthfront.magellan.lifecycle.LifecycleAwareComponent
+import com.wealthfront.magellan.lifecycle.lifecycle
+import com.wealthfront.magellan.lifecycle.lifecycleWithContext
+
+@Composable
+public fun Displayable(displayable: Displayable<View>, modifier: Modifier = Modifier) {
+  AndroidView(modifier = modifier, factory = {
+    if (displayable.view == null) {
+      throw IllegalStateException(
+        "View does not exist on ${displayable::class.java.simpleName}. " +
+          "Is it attached to the lifecycle?"
+      )
+    }
+    displayable.view!!
+  })
+}
+
+public class ComposeStepWrapper(
```

Comment:
nit: the naming of the classes is a bit confusing. But I can't think of better names. Something we might want to revisit later.